backends/views.py

```diff
@@ -24,4 +26,13 @@ def complete(request, *args, **kwargs):
         request.session[key] = backend_state
```

Comment:
Can you write a test covering this code?

Reply:
which code are you referring to here?

Reply:
This line, the social complete view code